scripts/resolve_deps/main.go

```diff
@@ -0,0 +1,99 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/solo-io/ext-auth-plugin-examples/pkg/checks"
+	"os"
+	"sort"
+)
+
+func main() {
+	if len(os.Args) != 3 {
```

Comment:
I'm still not getting this script to run successfully - the first time, it finishes and outputs a modified go.mod, but those modifications then cause all subsequent go invocations (either this script, or running tests, or anything else) to fail due to this error, indicating a replace statement that we need was removed:

```
➜ make resolve-deps
go run scripts/resolve_deps/main.go go.mod _glooe/dependencies
go: github.com/solo-io/go-utils@v0.14.2 requires
	helm.sh/helm/v3@v3.0.0 requires
	github.com/deislabs/oras@v0.7.0 requires
	github.com/docker/docker@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
make: *** [resolve-deps] Error 1
```

Reply:
That's weird. I'' give it a try

Reply:
I've tested it with ` make GLOOE_VERSION=1.3.1 build`

And the result is:
```
Successfully built 44662ec9cdd1
Successfully tagged gloo-ext-auth-plugin-sample:0.0.1
```